src/setup/rgw_start.py

```diff
@@ -28,8 +29,9 @@ def start_rgw(conf: MappedConf, config_file, log_file, index: str = '1',):
         """Start rgw service independently."""
         Log.info("Starting radosgw service.")
         try:
-            # TODO: To replace os.system with SimpleProcess calls
-            os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
+            #os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
+            cmd = f"/usr/bin/radosgw -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file} &"
+            result = SimpleProcess(cmd).run(shell=True)
```

Comment:
Why do we need Shell=True? Codacy flagging it too as security issue. Address other codacy issues too

Reply:
Removed this.